tests/spread/plugins/v2/snaps/conda-hello/snap/snapcraft.yaml

```diff
@@ -16,7 +16,7 @@ apps:
 parts:
   ipython:
     plugin: conda
-    conda-miniconda-version: 4.6.14
+    conda-miniconda-version: "py39_4.12.0"
```

Comment:
This version is mismatched with the conda-python-version.

Reply:
made 3.9 match too